static/chats.js

```diff
@@ -162,48 +165,51 @@ function saprotiKomandu(teksts) {
 }
 
 function uzstadiVaardu(jaunaisVards) {
-  const vecaisVards = vards
-  setCookie('name', jaunaisVards, 90)
-  vards = jaunaisVards
+  const vecaisVards = vards;
+  vards = jaunaisVards;
   return `${vecaisVards} kļuva par ${vards}`
 }
 
-function gStavoklis(teksts) {
-  let vardi = teksts.split(" ");
-  /* let izvads = vardi.slice(1); */
- let izvads = Object.assign({}, vardi.slice(1));
-  
-  fetch('/chats/set_mood', {
+function nolasiGarastavokli(gStavoklis){
+  const gStavoklisUzServeri = new GaraStavoklis(vards, gStavoklis.join(' '))
+  let parameters = {
     method: 'POST',
+    body: JSON.stringify({"mood": gStavoklisUzServeri}),
     headers: {
-        'Content-Type': 'application/json'
-    },
-    body: JSON.stringify({ "mood": izvads })
-});    
-
-gStavoklisNoFaila();
-
-}
-
-function gStavoklisNoFaila(){ 
-fetch('/chats/get_mood')
-.then(function(response) {
-  return response.json()
-})
-.then(function(newMood) {
-  console.log(newMood);
- let saturs = Object.values(newMood);
- console.log(saturs);
- let satursV = Object.keys(newMood);
- console.log(satursV);
- setCookie('mood', saturs, 90)
-})
+      'Content-Type': 'application/json'
+    }
+  }
+  fetch('/garastavoklis/pierakstit_grarastavokli', parameters)
+  .then(res => res.json())
+ .then((data) => {
+    console.log('Success:', data)
+  })
+  .catch((error) => {
+    console.error('Error:', error)
+  })
+  lasiGarastavokli()
+  uzstadiVaardu(vards+" "+gStavoklis)
+  
+  return ` tagad *${gStavoklis}*`
 }
 
 function paradiPalidzibu() {
-  return 'Pieejamās komandas : "/vards JaunaisVards", "/palidziba", "/versija"'
+  return 'Pieejamās komandas : "/vards JaunaisVards", "/palidziba", "/versija", "/es" '
 }
 
+class GaraStavoklis {
+  constructor(vards, gStavoklis, laiks) {
+    this.vaards = vards
+    this.garaStaavoklis = gStavoklis
+    this.precizsLaiks = laiks
```

Comment:
?
Man šķiet, ka var iztikt bez klases. + laiks ir ziņai, dēļ kam garastāvoklim? Mēs kaut kur to atrādīsim?